app/src/main/res/layout/view_focused_view.xml

```diff
@@ -23,6 +23,6 @@
     app:isExpandable="false"
     app:showPlaceholders="false"
     app:favoritesPlacement="focused_state"
-    android:background="?attr/daxColorSurface" />
+    android:background="?attr/daxColorBrowserOverlay" />
```

Comment:
This doesn't change anything in production as `daxColorBrowserOverlay == daxColorSurface` in production themes.